packages/create/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@sku-lib/create",
+  "version": "0.0.0",
+  "description": "Create sku projects",
+  "private": true,
+  "bin": {
+    "create-sku": "./dist/cli.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=18"
+  },
```

Comment:
This needs to align with `sku`'s config:
https://github.com/seek-oss/sku/blob/ac06eef4c8061d6b252339093a0d146c82844d4d/packages/sku/package.json#L36-L38